package.json

```diff
@@ -12,11 +13,17 @@
     "url": "http://tareksherif.net/"
   },
   "license": "MIT",
+  "scripts": {
+    "build": "browserify --standalone PicoGL -o build/picogl.js src/index.js && npm run compress",
+    "compress": "uglifyjs build/picogl.js --compress --mangle > build/picogl.min.js"
+  },
```

Comment:
Would it be possible to make these grunt tasks? In particular,  don't like the fact that two separate versions of uglify are dependencies. If you're not familiar with grunt, I can do that after the merge.